### PR TITLE
Rewards page, reward history and reward shop

### PR DESCRIPTION
**Pull request by aezouhri**
All tested need to implement api calls.

<!Pull Request Template>

## Description

<!Please provide a summary of the changes you have made. Include any relevant motivation and context. Mention any dependencies that are required for this change.>
Created the rewards page where users can see the total points they currently have accumulated. There is also 2 additional section, the history tab where users can look at their previous task completion as well as rewards claimed. ~~And the Shop tab where they can use their points.~~

You can access the rewards page at "/rewards".

## Tests
All elements and logic is being tested, however the reward.test.js only shows 20% coverage. That is due to the fact I am using a useEffect hook in a component rendered by the rewards page. 

## UI Changes
image

## Issues
#124 #119 #116 #117 #120 #123 
<!Mention any issues or bugs this PR addresses or fixes. Link any related issues.>

## Additional Notes

_Add any other information about the PR here._
Shop tab has been temporarily commented out until we figure out how to integrate it in our user-interaction.
---

### CODE REVIEW

**Comment by saanbe16** [comment]
This PR was not complete and still needed to use api calls

**Review by bdesollar — APPROVED** [review]
Looks good Adnane. Wait for @saanbe16 to approve before merging

File: web/src/components/Shop.js
Comment (saanbe16) [inline web/src/components/Shop.js:4]: @aezouhri, is this component currently being used?

**Review by saanbe16 — APPROVED** [review]
looks good

**Pull request merged**
